client/src/Leaderboard/components/NominatorRewardsList.tsx

```diff
@@ -80,6 +109,9 @@ const NominatorRewardsList = () => {
             onChange={onChange}
           />
         </div>
+        <div className='w-full flex mt-2 justify-center md:justify-end'>
```

Comment:
Can we add the lazy download button here in the desktop version:

<img width="1529" alt="image" src="https://github.com/subspace/astral/assets/5139220/d79d4c83-db00-41e5-a487-d4468edc991b">

So it's more harmonious.